lib/console/logs/provider/elastic.ex

```diff
@@ -50,6 +59,31 @@ defmodule Console.Logs.Provider.Elastic do
   end
   defp format_hits(_), do: []
 
+  defp format_aggregation_response(%Snap.SearchResponse{aggregations: aggs}) do
+    case aggs do
+      %{"logs_over_time" => %Snap.Aggregation{buckets: buckets}} ->
+        Enum.map(buckets, fn bucket ->
+          %{
```

Comment:
could you make a struct for this as well?  Will make it more straightforward to understand the output typing